cmd/api-endpoint/main_test.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+)
+
+// Use the common main for all tests in this package.
+func TestMain(suite *testing.M) {
+	cmd.CMDServerTestingMain(suite)
+}
+
+// Test to ensure an API endpoint that requires parameters works correctly.
+// Since all CMDs use the same infrastructure, only one endpoint needs to be tested.
```

Comment:
This sentence is not 100% true.
You have special cases for specific endpoints in your code.
I think those special cases should go away, but I would also drop this sentence.
Testing other endpoints would be useful later to try out other edge (but not hardcoded special) cases (like if we were sending a file).